tests/pyrog/migrations/test_0013_conditions_and_filters_sqlinput.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.mark.migration(app_label="pyrog", migration_name="0012_drop_input")
+def test_migrate(migrator, state, input_group_factory, column_factory, owner_factory):
```

Comment:
It's not a good idea to use factories here, it won't work anymore if we change them